src/Shared/Recruit.Shared.Web/Configuration/EnvironmentNames.cs

```diff
@@ -8,16 +8,14 @@ public static class EnvironmentNames
         public const string Development = "DEVELOPMENT";
         public const string AT = "AT";
         public const string TEST = "TEST";
+        public const string TEST2 = "TEST2";
         public const string DEMO = "DEMO";
         public const string PREPROD = "PREPROD";
         public const string PROD = "PROD";
 
-        public static string[] GetTestEnvironmentNames() => new []{ Development, AT, TEST, DEMO };
-        
-        public static string GetTestEnvironmentNamesCommaDelimited() => string.Join(",", Development, AT, TEST, DEMO);
-
-        public static string GetNonProdEnvironmentNamesCommaDelimited() => string.Join(",", Development, AT, TEST, DEMO, PREPROD);
-
+        public static string[] GetTestEnvironmentNames() => new []{ Development, AT, TEST, TEST2, DEMO };
```

Comment:
For complicated reason this method conflicts with the one below regarding what are "test" envs. One includes DEMO and the other doesn't and they want the GTM snippet to be output for the DEMO env. Suggestion is to remove DEMO from the `GetTestEnvironmentNames` method.

Reply:
Could `GetTestEnvironmentNamesCommaDelimited` be created from` GetTestEnvironmentNames`?

Reply:
The reason I dint use GetTestEnvironmentNames() is because it had Demo in it.